src/LinkedList.js

```diff
@@ -0,0 +1,129 @@
+class Node {
+  constructor() {
+    this.data
+    this.next = null
+  }
+}
+
+class LinkedList {
+  constructor() {
+    this.head = null
+    this.tail = null
+  }
+
+  getHeadNode() {
+    return this.head
+  }
+
+  getTailNode() {
+    return this.tail
+  }
+
+  contains(element) {
+    let elementFound = false
+    let iterator = this.head
+    while(iterator) {
+      if(iterator.data === element) elementFound = true
+      iterator = iterator.next
+    }
+    return elementFound
+  }
+
+  find(element) {
+    let iterator = this.head
+    while(iterator) {
+      if(iterator.data === element) return iterator
+      iterator = iterator.next
+    }
+    return -1
+  }
+
+  insert(element) {
+    let tail = new Node()
```

Comment:
It's a little confusing to call this `tail`, since it's not yet the tail when it's defined. It's also confusing because there's already a `this.tail` instance variable defined.

Try calling it `newNode` or something similar.